lib/rbtrace/cli.rb

```diff
@@ -407,6 +415,10 @@ def self.run
           tracer.puts "=> #{res}"
         end
 
+      elsif opts[:interactive_given]
+        require "rbtrace/interactive/#{opts[:interactive]}"
+        load `which #{opts[:interactive]}`.chomp
```

Comment:
I don't like shelling out either, would try to come up with the other way. However I don't think `File.expand_path(__FILE__)` would work here because it's not loading the file from `rbtrace` but the executable which installed by the other gems (or `irb`)